modules/ROOT/pages/dos-policy.adoc

```diff
@@ -1,89 +1,102 @@
 = Denial Of Service Policy
 
-The Denial of Service (DoS) policy prevents attackers from flooding your network to prevent legitimate network traffic to your APIs. +
-Malicious clients, for example, could send huge malicious payloads designed to consume resources and bandwidth.
+The Denial of Service (DoS) policy prevents attackers from flooding your network to prevent legitimate network traffic to your APIs. For example, malicious clients could send huge payloads designed to consume resources and bandwidth to your network.
 
-The DoS policy allows you to configure a time span in which you can catch specific error types, and set up the action to take for each type of error encountered. If the policy encounters more errors than your configured threshold coming from the same IP address, the policy can either drop the connection silently (doesn't attempt the TLS connection), or drop the connection immediately and return a `503` error.
+When you create a DoS policy, you configure a time span and action to take when the <<error_types,error types>> you configure are encountered. If the policy encounters more errors than your configured threshold coming from the same IP address, the policy can either drop the connection silently (doesn't attempt the TLS connection), or drop the connection immediately and return a `503` error.
 
-This source is IP based. If an attacker spoofs the source IP address, these measurements cannot prevent the attack.
+The DoS policy determines the source of the request in the following three ways:
 
+* TCP Message - source IP address 
+* Source IP address header field
+* Proxy Protocol IP address
+
+Since the source of the request is based on the IP address, if an an attacker spoofs the source IP address, the DoS Policy cannot prevent the attack.
+
+[[error_types]]
 == Error Types
 
+When you configure the DoS policy, you can configure different error types, which correspond to existing security policies. For example, the *Protocol Errors* correspond to HTTP Limits policy violations.  When you configure the different error types, you are configuring the threshold at which policy violations are sent (escalated) to the DoS policy for evaluation. See <<dos_thresholds_and_escalations,Rules, Thresholds, and Escalations>>. The way you configure the errors determines what action is taken when the configured threshold is met.  
+
 [%header%autowidth.spread,cols="a,a"]
 |===
 |Error |Description
 | *Protocol Errors*
 | When a client sends malformed protocol messages.
 
-A CAP policy violation escalates directly as a protocol error.
+An HTTP Limits policy violation is escalated to the DoS policy as *Protocol Errors*.
```

Comment:
Mysti said we're only supposed to use bold for UI fields and not for emphasis.

Reply:
Yes, these are UI. Each one is a tab.